tasklist/src/pages/listTask/listTask.html

```diff
@@ -18,6 +18,38 @@
 </ion-header>
 
 <ion-content>
+  <ion-item>
+    <ion-searchbar [(ngModel)]="searchName" (ionInput)="getDataFromApi()"></ion-searchbar>
+  </ion-item>
+  <ion-item>
+    <ion-label>Id</ion-label>
```

Comment:
The user don't know the `ID` and can't read it. You need to put the `ID` in the `list` and the `detail` of the task if you want filter by `ID`